topics/proteomics/tutorials/encyclopedia/data-library.yaml

```diff
@@ -0,0 +1,127 @@
+---
+destination:
+  type: library
+  name: GTN - Material
+  description: Galaxy Training Network Material
+  synopsis: Galaxy Training Network Material. See https://training.galaxyproject.org
+items:
+- name: Proteomics
+  description: Training material for proteomics workflows in Galaxy
+  items:
+  - name: EncyclopeDIA
+    items:
+    - name: 'DOI: 10.5281/zenodo.13505774'
+      description: latest
+      items:
+      - url: https://zenodo.org/record/13505774/files/191023JAT05_P_1ug_495_605_4_20.raw
+        src: url
+        ext: 
+        info: https://doi.org/10.5281/zenodo.13505774
+      - url: https://zenodo.org/record/13505774/files/191023JAT04_P_1ug_395_505_4_20.raw.mzml
+        src: url
+        ext: 
+        info: https://doi.org/10.5281/zenodo.13505774
+      - url: https://zenodo.org/record/13505774/files/191023JAT05_P_1ug_495_605_4_20.raw.mzml
+        src: url
+        ext: 
```

Comment:
@subinamehta can you maybe add the galaxy datatype here?

Reply:
should I add https://usegalaxy.eu in the url?

Reply:
No. The ext: should be the galaxy datatype.

Reply:
ok got it! I ll get it done!